torch/csrc/autograd/python_variable.cpp

```diff
@@ -154,10 +154,11 @@ PyObject *THPVariable_pynew(PyTypeObject *type, PyObject *args, PyObject *kwds)
   PyObject *grad_fn = NULL;
   char is_volatile = 0;
   char requires_grad = 0;
+  const char* pretty_name = NULL;
 
-  const char *accepted_args[] = {"data", "requires_grad", "volatile", "_grad_fn", NULL};
-  if (!PyArg_ParseTupleAndKeywords(args, kwds, "|ObbO", (char**)accepted_args,
-      &data, &requires_grad, &is_volatile, &grad_fn))
+  const char *accepted_args[] = {"data", "requires_grad", "volatile", "_grad_fn", "pretty_name", NULL};
```

Comment:
If it's going to become part of public API then please let's just call it "name"